src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1212,7 +1212,10 @@ def create_cohort(args):
     entity = rec_descriptor.model["global_primary_key"]["entity"]
     field = rec_descriptor.model["global_primary_key"]["field"]
     filters = resp.get("filters", {})
-    payload = cohort_final_payload(samples, entity, field, filters, from_project)
+    try:
+        payload = cohort_final_payload(samples, entity, field, filters, from_project)
+    except Exception as e:
+        err_exit("{}: {}".format(entity_result["id"], e))
```

Comment:
Does `format()` transform type `Exception` to `str` by default?

Reply:
Good catch. `str.format` should use the string value of the `Exception`
> ...Returns a copy of the string where each replacement field is replaced with the string value of the corresponding argument.
https://docs.python.org/3/library/stdtypes.html?highlight=str%20format#str.format
(also tested for Python 2 & 3)